backend/database/session.py

```diff
@@ -2,8 +2,19 @@
 from sqlalchemy.orm import sessionmaker
 from backend.api.config import settings
 
+
+def _get_database_url() -> str:
+    match settings.environment:
+        case "local":
+            return settings.database_url_sqlalchemy
+        case "ci" | "prod":
```

Comment:
Requires GitHub secret to be changed to `ci` for the "ETL to Neon" script to work.